### PR TITLE
Admin orders

### PR DESCRIPTION
**Pull request by antciccone**
-Added drop down button to view orders by their status 
-Wrote a test for the dropdown 
-One problem that I can't seem to fix is that the drop down menu stays down. I think it's something we can refactor at the end. let me know if any of you find a solution for that

### CODE REVIEW

**Comment by antciccone** [comment]
fixed merge conflict

File: app/models/orders_trip.rb [hunk 1]
```diff
@@ -9,4 +9,8 @@ class OrdersTrip < ApplicationRecord
   def subtotal
     trip_price * quantity
   end
+
+  def self.trip_quantity
+    pluck(:quantity).reduce(:+)
```
Comment (rtravitz) [inline app/models/orders_trip.rb]: Does the active record method sum work here?

File: spec/factories/factories.rb [hunk 2]
```diff
@@ -31,6 +31,7 @@
     password_confirmation "password"
   end
 
+
```
Comment (rtravitz) [inline spec/factories/factories.rb:34]: Extra space is adding another file change to the commit 🙁

File: spec/features/admin/a_admin_can_view_all_orders_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+
+  scenario "an admin can view orders by specific category" do
+    trip1, trip2 = create_list(:trip, 22)
+
+    admin_user = create(:user)
+    admin_user.admin!
+    user1, user2 = create_list(:user, 2)
+    user1.orders.create(cost: 12, status: "ordered")
+    user2.orders.create(cost: 12, status: "paid")
```
Comment (rtravitz) [inline spec/features/admin/a_admin_can_view_all_orders_spec.rb:35]: Tests look great! ✨

**Review by rtravitz — COMMENTED** [review]
Could you check out the sum thing?

File: app/models/orders_trip.rb (hunk 1 above)
Comment (antciccone) [reply app/models/orders_trip.rb]: ah good thought. ill change that

**Comment by rtravitz** [comment]
🙌

**Pull request merged**